boards/b-l072z-lrwan1/include/board.h

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @defgroup    boards_b-l072z-lrwan1 ST b-l072z-lrwan1 discovery
+ * @defgroup    boards_b-l072z-lrwan1 STM b-l072z-lrwan1 discovery
```

Comment:
Isn't the company called ST? Otherwise, in the file brief below it is also called ST